src/backend/distributed/metadata/node_metadata.c

```diff
@@ -1539,6 +1626,26 @@ EnsureCoordinator(void)
 }
 
 
+/*
+ * InsertPlaceholderCoordinatorRecord inserts a placeholder record for the coordinator
+ * to be able to create distributed tables on a single node.
+ */
+void
+InsertPlaceholderCoordinatorRecord(void)
```

Comment:
Would that make sense to add a NOTICE message in this method saying that the coordinator is added to the cluster so that users can keep this in mind. Otherwise one possibility is that users might run create_distributed_table etc by accident when there is no worker node and the coordinator will be added implicitly, then when they add the workers they might not be aware that coordinator is added as a worker(maybe they don't want this).

Reply:
makes sense

Reply:
added a notice to master_add_node (citus_add_node)